lib/core/src/lib/card-view/validators/card-view-item-match.valiator.ts

```diff
@@ -19,16 +19,17 @@ import { CardViewItemValidator } from '../interfaces/card-view.interfaces';
 
 export interface MatchValidatorParams {
     expression: string;
+    requiresMatch?: boolean;
     flags?: string;
 }
 
 export class CardViewItemMatchValidator implements CardViewItemValidator {
     message = 'CORE.CARDVIEW.VALIDATORS.MATCH_VALIDATION_ERROR';
 
-    constructor(private expression: string, private flags?: string) {}
+    constructor(private expression: string, private requiresMatch?: boolean, private flags?: string) {}
```

Comment:
Maybe it will be good idea to move that new parameter (requiredMatch) as last item of constructor? The reason is following: 
If any external user uses this class from ADF and initializes this class like: 
const validator = new CardViewItemMatchValidator(someExpression, someFlags); 

then this change will introduce breaking change for then because now at second parameter you have requiresMatch and not flags so they will get error that someFlags is string but boolean is expected. So we will enforce them to update their code. 

If you move requiresMatch at the end of constructor then everything will be fine for external customer because they won't need to update their code by changing order of parameters.

Reply:
very good explanation! fixed it and moved it as the last parameter